prbt_hardware_support/CMakeLists.txt

```diff
@@ -206,7 +206,7 @@ if(CATKIN_ENABLE_TESTING)
   if(ENABLE_COVERAGE_TESTING)
     include(CodeCoverage)
     APPEND_COVERAGE_COMPILER_FLAGS()
-    set(COVERAGE_EXCLUDES "*/${PROJECT_NAME}/test*" "*/devel/include/${PROJECT_NAME}/*")
+    set(COVERAGE_EXCLUDES "*/${PROJECT_NAME}/test*" "*/ModbusMsgInStamped.h")
```

Comment:
Please pose a separate PR for this.

Reply:
This line reverts dcc5ae1d1.
I had opened #121 as separate PR. Rebasing onto current melodic devel brought your change from dcc5ae1d1 in here. But if you set great value on it, I'll open another one-line PR, just let me know!

Reply:
I see